libr/include/r_util/r_asn1.h

```diff
@@ -7,6 +7,9 @@
 extern "C" {
 #endif
 
+#define MOVE_PTR(dst, src) { ((dst) = (src)); (src) = NULL; }
```

Comment:
the MOVE_PTR one can be generic and moved into r_types.h ... maybe R_PTR_MOVE() ?